client/src/components/Button.js

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+const Button = styled.button`
```

Comment:
The CSS of this button limits its text length.
If this is mandatory, it's fine. Otherwise, I would  not set a fixed width or use min-width instead.